specs/trust-store-trust-policy.md

```diff
@@ -414,10 +411,11 @@ When delta CRLs are implemented, the following results can occur during revocati
 
 ##### OCSP Download
 
-OCSP URLs can be obtained from the certificate's authority information access (AIA) extension as defined in [RFC-2560](https://datatracker.ietf.org/doc/html/rfc2560).
+OCSP URLs can be obtained from the certificate's authority information access (AIA) extension as defined in [RFC-6960](https://datatracker.ietf.org/doc/html/rfc6960).
+If the certificate contains multiple OCSP URLs, then each URL is invoked in sequential order.
 Notary V2 will wait for a default threshold of 5 seconds to receive an OCSP response.
-If OCSP response is not available within the timeout threshold the revocation result will be "revocation unavailable".
 The user may be able to configure this threshold.
+If OCSP response is not available within the timeout threshold the revocation result will be "revocation unavailable".
```

Comment:
I am updating the threat model for OCSP. If OCSP responder is not accessible, could it be a DoS? Since the revocation status is always unavailable, so verify will always fail. If we disable the revocation check, we are at the risk of using an invalid certificate for verification.

Reply:
Yes, it is indeed possible for a Denial of Service (DoS) attack on the OCSP endpoint to render the OCSP server inaccessible. In the scenario where the revocation verification level is set to "enforce", this will result in a signature verification failure. However, for other cases (log or skip), the signature verification will succeed.